packages/docs/README.md

```diff
@@ -0,0 +1,76 @@
+#TODO
+@async
+@author
+@ignore
+@see
+@summary
+
+# docs
+
+In order for comments to be processed by the docs system they must begin with \*
```

Comment:
you mean `/*`? not `\*`

Also, this is just called "block comments" btw, if you wanna use the nomenclature

Reply:
The syntax is actually /** */ (Notice the extra asterix at the start). This is actually what I meant here.